client/components/plans/plans-skip-button/README.md

```diff
@@ -0,0 +1,36 @@
+# Plans Skip Button.
+
+Render a button that allows a user to forego (paid) plans selection during creation of a new site, and use the Free plan instead.
+
+## Props 
+
+* **isRtl** — (optional) Whether the button should be rendered for a Right-to-Left locale.
```

Comment:
It's unfortunate that devs need to remember to pass this to get proper behavior. Maybe worth connecting the component, definitely worth investigating a better solution.